src/__tests__/index.browser.js

```diff
@@ -28,10 +37,18 @@ test('hydration from element', t => {
   translations.setAttribute('type', 'application/json');
   translations.setAttribute('id', '__TRANSLATIONS__');
   translations.textContent = JSON.stringify(hydrationState);
+  // $FlowFixMe
```

Comment:
Why FlowFixMe? Do we need to just add a check that translations is an HTMLElement?

Reply:
Good call!  In this case, `translations` is an `HTMLScriptElement` which extends an `HTMLElement` so that type checking should actually work out here.  The error being ignored is instead:

```
[flow] Cannot call `document.body.removeChild` because property `removeChild` is missing in null [1]. (References: [1])
```

Implying that `document.body` may be `null`.  I'll update this to add a check for `document.body` to fix this since this is super quick.

As as a bit more context, my first order concern is to maximize coverage of:
* Any missing `fusion-core` export APIs
* Token types
* Plugin export APIs

This means unit tests may end up having a lot more ignored errors so that I can test the exports but skip over other things (like `document.body`) to try to move more quickly.  It's not ideal, but I think the right trade-off to get core Flow types into the hands of consumers more quickly.